code/modules/mob/living/silicon/ai/ai_say.dm

```diff
@@ -74,21 +74,23 @@ GLOBAL_VAR_INIT(announcing_vox, 0) // Stores the time of the last announcement
 	set desc = "Display a list of vocal words to announce to the crew."
 	set category = "AI Commands"
 
-	var/list/dat = list()
+	var/static/string_dat
+	if(!string_dat)
+		var/list/dat = list()
 
-	dat += "Here is a list of words you can type into the 'Announcement' button to create sentences to vocally announce to everyone on the same level at you.<BR> \
-	<UL><LI>You can also click on the word to preview it.</LI>\
-	<LI>You can only say 30 words for every announcement.</LI>\
-	<LI>Do not use punctuation as you would normally, if you want a pause you can use the full stop and comma characters by separating them with spaces, like so: 'Alpha . Test , Bravo'.</LI></UL>\
-	<font class='bad'>WARNING:</font><BR>Misuse of the announcement system will get you job banned.<HR>"
+		dat += "Here is a list of words you can type into the 'Announcement' button to create sentences to vocally announce to everyone on the same level at you.<BR> \
+		<UL><LI>You can also click on the word to preview it.</LI>\
+		<LI>You can only say 30 words for every announcement.</LI>\
+		<LI>Do not use punctuation as you would normally, if you want a pause you can use the full stop and comma characters by separating them with spaces, like so: 'Alpha . Test , Bravo'.</LI></UL>\
+		<font class='bad'>WARNING:</font><BR>Misuse of the announcement system will get you job banned.<HR>"
 
-	// Show alert and voice sounds separately
-	var/vox_words = GLOB.vox_sounds - GLOB.vox_alerts
-	dat += help_format(GLOB.vox_alerts)
-	dat += "<hr>"
-	dat += help_format(vox_words)
+		// Show alert and voice sounds separately
+		var/vox_words = GLOB.vox_sounds - GLOB.vox_alerts
+		dat += help_format(GLOB.vox_alerts)
```

Comment:
Caching this wont work as its tied to each AI mob's UID to be able to send things. 

While the amount of games we have 2 AIs in a round are uncommon, it can still happen.

Reply:
Whoops, thanks for pointing that out. I think I have a solution to this

Reply:
Cached per AI now. That's one hell of a long string

Reply:
I would add the cache var to the unable to VV get list to make it so you can look at the VV UI without dying

Reply:
done